src/wallet/rpc/addresses.cpp

```diff
@@ -541,6 +541,10 @@ RPCHelpMan getaddressinfo()
                         {
                             {RPCResult::Type::STR, "label name", "Label name (defaults to \"\")."},
                         }},
+                        {RPCResult::Type::ARR, "use_txids", "",
```

Comment:
Are there any guidelines when a response attribute should be optional vs always present?